rcon/steam_utils.py

```diff
@@ -18,8 +19,12 @@
 @ttl_cache(60 * 60 * 24, cache_falsy=False, is_method=False)
 def get_steam_profile(steamd_id):
     profiles = get_steam_profiles([steamd_id])
+
     if not profiles or len(profiles) == 0:
         return None
+    
+    time_played = get_hll_playtime_in_hours(steam_id=steamd_id)
```

Comment:
You could test if the profile is public here first, that'd avoid making an API call for nothing. We need to be mindful of the time delays we add to the even loop as it's been growing a lot already.